internal/helmdeployer/helmcache/secret.go

```diff
@@ -0,0 +1,76 @@
+package helmcache
+
+import (
+	"context"
+
+	corecontrollers "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	applycorev1 "k8s.io/client-go/applyconfigurations/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type SecretClient struct {
+	cache     corecontrollers.SecretCache
```

Comment:
Is there any chance to use [`client-go`'s cache](https://pkg.go.dev/k8s.io/client-go/tools/cache) instead of this one? (in order to not mix clients).
On the other hand, I do agree this implementation is pretty clean.

Reply:
Wrangler and lasso are using client-go `ShareInformer` for the cache. I didn't want to create another client-go informer as the wrangler one is already created, and it is used in more places.